public/app/plugins/datasource/tempo/datasource.ts

```diff
@@ -378,6 +399,60 @@ export class TempoDatasource extends DataSourceWithBackend<TempoQuery, TempoJson
     };
   }
 
+  handleMetricsSummary = (target: TempoQuery, query: string, options: DataQueryRequest<TempoQuery>) => {
+    const groupBy = target.groupBy
+      ?.filter((f) => f.tag)
+      .map((f) => {
+        if (f.scope === TraceqlSearchScope.Unscoped) {
+          return `.${f.tag}`;
+        }
+        return f.scope !== TraceqlSearchScope.Intrinsic ? `${f.scope}.${f.tag}` : f.tag;
+      })
+      .join(', ');
+
+    return this._request('/api/metrics/summary', {
+      q: query,
+      groupBy,
+      start: options.range.from.unix(),
+      end: options.range.to.unix(),
+    }).pipe(
+      startWith({
+        data: [],
+        state: LoadingState.Loading,
+      }),
+      map((response) => {
+        if (response.state !== LoadingState.Loading && !response.data.summaries) {
```

Comment:
Why the `response.state !== LoadingState.Loading`? Is that just bucause we have that `startWith` value?